ethereum_history_api/circuits/lib/src/receipt.nr

```diff
@@ -33,9 +33,9 @@ struct ForeignCallTxReceiptPartial<LOG_NUM, MAX_LOG_DATA_LEN> {
     logs_bloom: [u8; BLOOM_FILTER_LEN]
 }
 
-struct TxReceiptProof {
+struct TxReceiptProof<MAX_RLP_LEN> {
```

Comment:
Why `MAX_RLP_LEN` not `MAX_PROOF_LEN` for consistency?

Reply:
proof len and rlp len are different things. rlp len is for max rlp value len and proof is for whole proof